backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,77 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-kubevirt"]
+// If docs team approves the name, "Configuring OADP with Kubevirt" great but I suspect it will need to be "Configuring OADP with OpenShift Virtualization" or "Configuring OADP with Virtualization" or perhaps... bear with me.. "OADP and OpenShift-Virtualization"
+= Configuring the OpenShift API for Data Protection with {VirtProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-kubevirt
+:installing-oadp-kubevirt:
+:credentials: cloud-credentials
+:provider: kubevirt
+
+toc::[]
+
+You install the {oadp-first} with {VirtProductName} by installing the OADP Operator and configuring a backup location. Then, you install the Data Protection Application.
+
+// include::snippets/oadp-mtc-operator.adoc[]
+
+Back up and restore virtual machines by using the xref:../../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)].
+
+include::snippets/oadp-virtualization-supported-backup-types.adoc
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Install the xref:../../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#about-installing-oadp[OADP Operator] according to the instructions for your storage provider.
+
+. Install the xref:oadp-installing-dpa-1-3_installing-oadp-kubevirt[Data Protection Application] with the `kubevirt` and `openshift` xref:../../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[plugins].
+
+. Back up virtual machines by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)].
```

Comment:
There will probably be sections added post-GA

Reply:
Please can I come back next week after GA and fix this

Reply:
I checked on this with the rest of the peer review squad, and the squad agrees merging this is not okay because it does not fit with mod docs standards for assemblies: https://redhat-documentation.github.io/modular-docs/#assembly-guidelines

Assemblies can have introductions, prerequisites, and modules as the main pieces. If you have a procedure, it belongs in a procedure module. I'm not sure what other post-GA sections would validate having a procedure in an assembly 🤷‍♀️ because they just aren't allowed.